src/server/database/Database/MySQLConnection.cpp

```diff
@@ -227,7 +227,7 @@ bool MySQLConnection::Execute(PreparedStatementBase* stmt)
 
     uint32 _s = getMSTime();
 
-#if MYSQL_VERSION_ID >= 80300
+#if !defined(MARIADB_VERSION_ID) && MYSQL_VERSION_ID >= 80300
```

Comment:
Do you want to add (...) to the second condition, too? 🙂

Reply:
> Do you want to add parantheses (...) to the second condition? 🙂

Of course.

Reply:
```c++
#if (!defined(MARIADB_VERSION_ID) && (MYSQL_VERSION_ID >= 80300))
```
Shouldn't it be this way?